app/library/postgres.py

```diff
@@ -149,6 +149,13 @@ async def delete_asset_from_postgres(asset_id: str, session):
             Key=asset_obj.s3_key,
         )
 
+        pos_key = asset_obj.screenshot_url.find("image/") + 6
```

Comment:
Ah, the `image/` is actually part of the key in S3. So the actual key for objects would be like `image/abcd.png`. So you don't need to do + 6 here